Makefile

```diff
@@ -228,21 +231,28 @@ deploy:
 	
 	@echo '==> Building...'
 	yarn
+	yarn nx reset
+	yarn lerna bootstrap
+	yarn lerna run build
```

Comment:
After some digging, I've now realized that this isn't the right place for these commands.

In this Makefile, we only run `yarn run tsc -b` to transpile `buildAndDeploySite.ts`, which is node script that `yarn buildAndDeploySite` calls.

That script ssh's to the remote server, rsyncs the directory over, and then runs [these scripts](https://github.com/owid/owid-grapher/blob/master/baker/Deployer.ts#L217) in sequence. 

So if anywhere's missing some extra lerna commands, it's there. The reason it had been working till this point is because we [were rsync'ing over locally built copies of the lerna libraries](https://github.com/owid/owid-grapher/blob/master/baker/Deployer.ts#L69) so it didn't need to rebuild them (though this would 100% introduce errors if you changed a lerna package on a different branch, built the packages, then switched back to master without rebuilding them.)

Posting the rest of my memory allocation notes in slack.